libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -470,7 +470,6 @@ - (NSArray*) getDevSupportInfos
             @"SDK Version", SALESFORCE_SDK_VERSION,
             @"App Type", [self getAppTypeAsString],
             @"User Agent", self.userAgentString(@""),
-            @"Browser Login Enabled", userAccountManager.advancedAuthConfiguration != SFOAuthAdvancedAuthConfigurationNone ? @"YES" : @"NO",
```

Comment:
We should still report whether browser login was used or not.

Reply:
+1

Reply:
@wmathurin The reason I took this out is because mdm settings will show it anyway. Do you still want to call it out?

Reply:
It could also come from the org settings. I would still have it called out.